contracts/test/bridge/artifacts-test/Proxy.json

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
I think this should be available directly from here: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/ERC1967/ERC1967Proxy.sol

We want to avoid using static artifacts as much as we can. Instead add a NPM dependency if not available, load the artifact from there and deploy.